sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -170,6 +171,34 @@ class VersionsSuite extends SparkFunSuite with Logging {
       client.createDatabase(tempDB, ignoreIfExists = true)
     }
 
+    test(s"$version: create/get/alter database should pick right user name as owner") {
+      if (version != "0.12") {
+        val currentUser = UserGroupInformation.getCurrentUser.getUserName
+        val ownerName = "SPARK_29425"
+        val db1 = "SPARK_29425_1"
+        val db2 = "SPARK_29425_2"
+        val ownerProps = Map("ownerName" -> ownerName)
+
+        // create database with owner
+        val dbWithOwner = CatalogDatabase(db1, "desc", Utils.createTempDir().toURI, ownerProps)
+        client.createDatabase(dbWithOwner, ignoreIfExists = true)
+        val getDbWithOwner = client.getDatabase(db1)
+        assert(getDbWithOwner.properties("ownerName") === ownerName)
+        // alter database without owner
+        client.alterDatabase(getDbWithOwner.copy(properties = Map()))
+        assert(client.getDatabase(getDbWithOwner.name).properties("ownerName") === currentUser)
```

Comment:
so the owner gets reset to `currentUser`. Is it expected?

Reply:
yes, it is. here we explicitly erase the owner here, in `ALTER DB` cmd the `properties` will be `originalDb.properties ++ Map()`

Reply:
then can we add a test to show this behavior in `HiveDDLSuite`?

Reply:
I can add a test for current behavior in `HiveDDLSuite `, because if we change ownerName/Type to private, create database/alter database cmd shall force to use spark's default user for lack of ownership switch syntax

Reply:
we can change this behavior later.